### ISSUE TITLE
Zoom and Highlight on the graph

### ISSUE THREAD

**Issue by MeyssamMolaei** [AykutSarac/jsoncrack-vscode#44]
### Feature

I want to suggest two feature on this useful extension,
1- Zoom feature: With big or complected JSON s, the tree is showing very small and its useless. So a zoom option should be a good.
2- Highlight: Once the developers are clicking on a pat of their JSON code, the related part can be highlight.

**Comment by pszNicx** [comment]
I agree. I have some pretty large elaborate JSON files and without a way to zoom or focus I can't really use this. I love the idea but unfortunately it doesn't work for me.

**Comment by louis-l** [comment]
+1

**Comment by peterkassner** [comment]
+3

**Comment by astr0gator** [comment]
@peterkassner Super helpful, thanks. Cmd + scroll would be very natural to have for mac users.
